config/locales/client.en.yml

```diff
@@ -5547,6 +5547,14 @@ en:
           dropdown: "Dropdown"
           upload: "Upload a file"
           multiselect: "Multiple choice"
+        edit_category:
+          toggle_freeform: "form template disabled"
+          toggle_form_template: "form template enabled"
+          select_template: "Select form templates"
+          select_template_help:
+            info: "Create or edit form templates at"
+            path: "Customize / Form Templates"
```

Comment:
Another small one, we should interpolate strings here, so translators can better understand the context. 

So this should be one string, something like this: 

```
"Create or edit templates at <a href='%{basePath}/admin/...'>Customize / Form Templates</a>"
```

Or, even simpler, we could keep the `LinkTo` helper and just add the full text in the anchor ("Add/Edit Form Templates"). 

Note also that not everybody that has access to the category edit screen will have access to the admin UI. Probably good to wrap this in a `isStaff` check for now.